index.js

```diff
@@ -1,3 +1,4 @@
-
+//cat's are always hungry
+var now = new Date();
 // the cat says
-console.log('Feed the kitties now!');
+console.log('Feed the kitties at ' + now);
```

Comment:
A little trick with console.log is that if you comma-separate the values it will automatically add the space: `console.log('Feed the kitties at', now)`